tests/Fixer/Strict/StrictParamFixerTest.php

```diff
@@ -155,6 +155,11 @@ public function __construct($foo, $bar) {}
         array_keys($foo, $bar);
     }',
             ],
+            [
+                '<?php
+    use function Foo\base64_decode;
+    foo($foo);',
```

Comment:
Can we have here `baz` instead of `foo` to point that function's name don't have to be the same as namespace?

Reply:
I reduced the test case to it's simplest form:
```php
<?php
use function \base64_decode;
foo($bar);',
```